web/modules/custom/profile_block/src/Controller/CustomModalController.php

```diff
@@ -25,22 +24,44 @@ class CustomModalController extends ControllerBase {
    */
   private $connection;
 
+  /**
+   * The current route match.
+   *
+   * @var \Drupal\Core\Routing\CurrentRouteMatch
+   */
+  protected $currentRouteMatch;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManager;
+
   /**
    * To initiate the database.
    *
    * @param \Drupal\Core\Database\Connection $connection
    *   The database connection to be used.
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $currentRouteMatch
+   *   The current route match.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(Connection $connection) {
+  public function __construct(Connection $connection, CurrentRouteMatch $currentRouteMatch, EntityTypeManager $entity_type_manager) {
     $this->connection = $connection;
+    $this->currentRouteMatch = $currentRouteMatch;
+    $this->entity_type_manager = $entity_type_manager;
```

Comment:
entity_type_manager should be entityTypeManager

Reply:
fixed here: [web/modules/custom/profile_block/src/Controller/CustomModalController.php](https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-a8c43901147783f211bc5d3e20d8b1634bd35b87941fb640c22e89af8a2765d5)